apps/s_server.c

```diff
@@ -1353,24 +1409,23 @@ int s_server_main(int argc, char *argv[])
             no_resume_ephemeral = 1;
             break;
         case OPT_PSK_IDENTITY:
-#ifndef OPENSSL_NO_PSK
             psk_identity = opt_arg();
-#endif
             break;
         case OPT_PSK_HINT:
 #ifndef OPENSSL_NO_PSK
```

Comment:
The identity_hint arg is only relevant to <=TLSv1.2 so the guard can stay. The identity arg is relevant to both TLSv1.2 PSK and TLSv1.3 so the guard is removed.